v2/internal/coord/coordinator.go

```diff
@@ -567,15 +574,13 @@ func (c *Coordinator) Bootstrap(ctx context.Context, seeds []kadt.PeerID) error
 
 // NotifyConnectivity notifies the coordinator that a peer has passed a connectivity check
 // which means it is connected and supports finding closer nodes
-func (c *Coordinator) NotifyConnectivity(ctx context.Context, id kadt.PeerID) error {
+func (c *Coordinator) NotifyConnectivity(ctx context.Context, id kadt.PeerID) {
```

Comment:
Why remove error from here but not from NotifyNonConnectivity? Let's remove from both

Reply:
Oh of course, missed that, thanks!

Reply:
(this is actually unused)